pkg/tracing/otlp/config_yaml.go

```diff
@@ -31,6 +31,8 @@ type Config struct {
 	RetryConfig        retryConfig       `yaml:"retry_config"`
 	Headers            map[string]string `yaml:"headers"`
 	TLSConfig          exthttp.TLSConfig `yaml:"tls_config"`
+	SamplerName        string            `yaml:"sampler_name"`
+	SamplerArg         string            `yaml:"sampler_arg"`
```

Comment:
Maybe we can try to keep field names similar to jaeger?
```suggestion
	SamplerType        string            `yaml:"sampler_type"`
	SamplerParam       string            `yaml:"sampler_param"`
```

Reply:
Doing that in my next commit!

Reply:
I'm a bit torn on this one, because OTEL uses slightly different terminology, i.e. sampler type from Jaeger and sampler name from OTEL do not align 1:1. I also assume not that far in the future we might deprecate the Jaeger client in favor of OTEL, so maybe it would make sense to stick with OTEL terminology? WDYT @saswatamcode?